src/anchorfm-pupeteer/index.js

```diff
@@ -79,6 +79,36 @@ async function selectDayInDatePicker(page, navigationPromise, day) {
   await navigationPromise;
 }
 
+async function clickSelector(page, selector, timeout) {
+  try {
+    await page.waitForSelector(selector, { timeout: timeout });
+    const element = await page.$(selector);
+    await clickDom(page, element);
+  } catch (error) {
+    console.error(`Error clicking selector: ${selector}`);
+    console.error(error);
+  }
+}
+
+async function clickXpath(page, xpath, timeout){
+  try{
```

Comment:
I don't think the try catch is necessary(same commend applies for the other method)